### ISSUE TITLE
Scheduled test runs fail after #523 was merged

### ISSUE THREAD

**Issue by mstenta** [farmOS/farmOS#524]
It seems that after #523 was merged MariaDB tests have been failing - although somewhat inconsistently. Opening this issue to start tracking clues...

Example failed run: https://github.com/farmOS/farmOS/actions/runs/2172632522

**Comment by mstenta** [comment]
Ah actually there may be multiple issues here... just noticing that all of the daily scheduled runs of `delivery.yml` have failed since #523 was merged, but NOT because of MariaDB tests failing... because the Docker image isn't building.

- https://github.com/farmOS/farmOS/actions/runs/2166039895
- https://github.com/farmOS/farmOS/actions/runs/2171643823
- https://github.com/farmOS/farmOS/actions/runs/2176050668
- https://github.com/farmOS/farmOS/actions/runs/2179363611
- https://github.com/farmOS/farmOS/actions/runs/2182971328

I bet this has to do with the way `FARMOS_VERSION` is being set now... it's probably not working correctly with the `scheduled` trigger.

This is different than the MariaDB failures, but I'll keep them in the same issue for now and generalize the title:

**Comment by mstenta** [comment]
> I bet this has to do with the way `FARMOS_VERSION` is being set now... it's probably not working correctly with the `scheduled` trigger.

Yup, the logic used to look like this:

```
      - name: Set FARMOS_VERSION for push event.
        if: ${{ !github.event.pull_request }}
        run: |
          echo "FARMOS_VERSION=${GITHUB_REF:11}" >> $GITHUB_ENV
          echo "FARMOS_REPO=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
```

That covered setting the `FARMOS_REPO` and `FARMOS_VERSION` environment variables in all contexts except pull requests earlier.

Now we explicitly set `FARMOS_REPO` and `FARMOS_VERSION` ONLY for branch/tag pushes and pull requests. But not for scheduled runs... that should be easy enough to fix...

**Comment by mstenta** [comment]
Renaming this to be specifically about scheduled runs. I'll create a separate issue for the MariaDB failures...

**Comment by mstenta** [comment]
I pushed a commit to the `2.x` branch of my fork that should fix this. I will wait to see if the scheduled run works tomorrow morning...

https://github.com/mstenta/farmOS/commit/65ace22777077b7d6ec4ba3f0c0ed27f94865e62

**Comment by mstenta** [comment]
That seems to have fixed it. Pushing this to origin...
